src/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfiguration.tsx

```diff
@@ -89,27 +91,23 @@ const isManagedNetworkingDisabled = (
     openshiftVersion &&
     cpuArchitecture === CpuArchitecture.ARM &&
```

Comment:
Is `ARM64_ARCHITECTURE_WITH_CLUSTER_MANAGED_NETWORKING` still a relevant FeatureId?

Is there not in the new API a generic FeatureId which can be used for whaterver the current `cpuArchitecture` is?
Maybe don't deal with this now, but IMO there should be a ticket to address this issue and other similar in the future.

Reply:
Ok. I've opened a ticket to track this cases: https://issues.redhat.com/browse/MGMT-14183

Reply:
Since the (old) features-support-level API has already been marked as deprecated, let's use MGMT-14183 to deal with its deprecation but in the UI and consider the removal of `ARM64_ARCHITECTURE_WITH_CLUSTER_MANAGED_NETWORKING` a sub-task inside the same issue.